src/PostgREST/App.hs

```diff
@@ -206,8 +204,13 @@ app dbStructure conf apiRequest =
         else return notFound
 
     (ActionRead, TargetRoot, Nothing) -> do
-      body <- encode <$> H.query schema accessibleTables
-      return $ responseLBS status200 [jsonH] $ cs body
+      let encodeApi ti = encodeOpenAPI ti host port
```

Comment:
host and port here are the ones from command line which are used to define interface/port to bind to
usually that is going to be an ip and usually, postgrest is going to sit behind a proxy (nginx) and the api will be accessible using a nice url (https://mydomain.com/api/v1).
When you are going to request the openapi schema, it's not going to match the public url, it's gonna say `192.168.10.10:3000`